src/Sylius/Bundle/PromotionBundle/test/src/Entity/PromotionSubject.php

```diff
@@ -11,15 +11,21 @@
 
 declare(strict_types=1);
 
-namespace AppBundle\Entity;
+namespace App\Entity;
 
 use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
 use Sylius\Component\Promotion\Model\PromotionInterface;
 use Sylius\Component\Promotion\Model\PromotionSubjectInterface;
 use Sylius\Resource\Model\ResourceInterface;
 
+#[ORM\Entity]
```

Comment:
Checking just in case, do we use attribute mappings in sylius/sylius?

Reply:
No we don't always XML mapping

Reply:
Yes, but that's normal that Sylius uses external configuration files. but in test application, that's better to use sth easier and more conventional.

Reply:
P.S. tests annotation usages reported https://github.com/Sylius/Sylius/issues/18030

Reply:
Already fix in a next PR cause we need to remove annotations. I'll mention your issue there.